stock_picking_filter/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module does the following:
+- Adds partner_invoice_id field to stock picking.
```

Comment:
Please add more background info or context wrt why this customization is needed.

Reply:
I've updated it but may adjust it further.